test/main.js

```diff
@@ -24,6 +26,8 @@ var SelfInsertingView = View.extend({
     render: function () {}
 });
 
+viewCompliance.view(test, ViewSwitcher, { view: new makeTestView()() });
```

Comment:
@RickButler I think I've spotted the error:

```
{ view: new makeTestView()() }
```

should be

```js
{ view: new (makeTestView())() }
```

I believe it was being interpreted as 

```js
var $ = new makeTestView();
var $$ = foo();

{ view: $$ }
```

Simple code typo. I wonder why it's passing on your machine. Anyway, fix that up, and you get a +1 from me.